benefits/locale/es/LC_MESSAGES/django.po

```diff
@@ -684,6 +684,19 @@ msgstr "Será dirigido a nuestro socio de pagos, "
 msgid "We don’t store your information, and you won’t be charged."
 msgstr "No almacenamos su información y no se le cobrará."
 
+msgid ""
+"The next step is to enroll the contactless card you will use to tap to ride "
+"for a reduced fare."
+msgstr ""
+"El siguiente paso es inscribir la tarjeta de pago sin contacto que usará "
```

Comment:
Did you write these Spanish translations?

Reply:
I did, but if we have a different workflow I can make changes.

Reply:
No that's great! We send all the English strings to a service to get them translated, so we typically add Spanish in later once we have them. We'll just need to double check these line up with what the service gives us, since we want Ditto/Figma to be the record of copy.

Reply:
Oh I see, that sounds good!

Reply:
Yes we typically do one big pass on the translations once we have them, so we'll come back to it for sure.

I'd say you can merge this now.

Reply:
Sounds good!